js/app.js

```diff
@@ -0,0 +1,58 @@
+// Creamos una función para cifrar cualquier palabra o frase
+function cipher() {
+// Creamos una variable input que almacene los datos que ingresan por medio de prompt
+  var input = prompt('Ingrese una frase:');
+  // Creamos una variable que tenga números ya que en el enunciado nos piden que no ingresen numeros y esto  nos va a servir como referencia
+  var num = (0, 1, 2, 3, 4, 5, 6, 7, 8, 9);
+  // Creamos una condicional para el caso de dejar espacio te reite que debes ingresar contenido
+  while (input === '') {
+    input = prompt('Ingrese una frase:');
+    // Despues creamos un ciclo para que pase por cada dato de la variable
+    for (var j = 0; j < input.length; j++) {
+    // La condicional que creamos es para verificar que el dato que se ingresa no es ningun número
+      while (input[j] === num) {
```

Comment:
Lamentablemente este `while` no hace lo que quieres que haga ya que no se pueden comparar un caracter con un array de esta manera. Para ver si un array contiene un valor, puedes usar `numbers.include(input[j])`.
Ahora, cuando encuentras un número y pides otro input, necesitas volver a correr todo el for() de la línea 11, no solo seguir comparando desde la posición `j`, ya que el valor de input ha cambiado.